_posts/2021-06-08-moveit-vs-moveit2.md

```diff
@@ -23,6 +23,16 @@ With the release of ROS 2 Galactic, we would like to share the new features only
     </tr>
   </thead>
   <tbody>
+    <tr>
+      <td>ROS1 Support</td>
+      <td class="done">✓</td>
+      <td class="dev">via ros1_bridge</td>
+    </tr>
+    <tr>
+      <td>ROS2 Support</td>
+      <td class="not">✕</td>
+      <td class="done">✓</td>
+    </tr>
```

Comment:
This feels a bit obvious. Should we just add a sentence that says something like: `While MoveIt was built using the original ROS ecosystems, MoveIt 2 had been engineered from the ground up to work with ROS 2`